src/components/m-header-logo/_template.js

```diff
@@ -1,7 +1,8 @@
 import html from 'nanohtml';
 import raw from 'nanohtml/raw';
 
-export default ({ src, alt = 'AXA Logo', href = '#' } = {}) => html`
+export default ({ src, alt = 'AXA Logo', href = '#' } = {
```

Comment:
This auto-fix isn't nice too

Reply:
Well, the linter configuration was set like that.

Reply:
Touche :)